gatsby-config.js

```diff
@@ -80,6 +80,23 @@ module.exports = {
       : []),
     'gatsby-plugin-typescript',
     'gatsby-plugin-emotion',
+    ...(process.env.GTM_TRACKING_ID
+      ? [
+          {
+            resolve: 'gatsby-plugin-google-tagmanager',
+            options: {
+              id: process.env.GTM_TRACKING_ID,
+              includeInDevelopment: false,
+              routeChangeEventName: 'e_routeChange',
```

Comment:
nice, I will chat to Lloyd to hear if if this should be e_pageView or what he prefers but we can leave as is for now.

Reply:
This will probably tell us if the route goes to 404 also
we can check to see if 404 'e_httpError'  event is needed